### PR TITLE
CI: Add release workflow for staging the Python package through GHA to PyPI

### PR DESCRIPTION
**Pull request by amotl**
## About
A release workflow for staging the Python package through GHA to PyPI.
Along the lines, the patch includes "developer guide" documentation for both Python and JavaScript.

### CODE REVIEW

File: DEVELOP.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    npm test
+
+Run specific tests:
+
+    ???
+
+Format code:
+
+    ???
```
Comment (amotl) [inline DEVELOP.md:69]: Didn't know relevant incantations for the JavaScript domain. Are there any?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        run: |
+          python -m pip install build twine
+          python -m build
+          twine check dist/{*.tar.gz,*.whl}
+
+      - name: Publish package to PyPI
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (amotl) [inline .github/workflows/release.yml]: The GHA workflow needs the `PYPI_API_TOKEN` secret, which has just been added to the repository.

File: DEVELOP.md (hunk 1 above)
Comment (surister) [reply DEVELOP.md:69]: Ah sorry I didn't see the "?", no currently there is no `format`, maybe we should add it int he future with `prettier`, for running a single test is `npm test -- other.test.js`

**Review by surister — APPROVED** [review]

File: DEVELOP.md (hunk 1 above)
Comment (amotl) [reply DEVELOP.md:69]: Thanks. Yeah, please fill in the gaps, or delete sections where applicable.

**Pull request merged**